src/st-info/info.c

```diff
@@ -38,9 +37,6 @@ static void stlink_print_info(stlink_t *sl) {
     printf("  flash:      %u (pagesize: %u)\n", (uint32_t)sl->flash_size, (uint32_t)sl->flash_pgsz);
     printf("  sram:       %u\n", (uint32_t)sl->sram_size);
     printf("  chipid:     0x%.3x\n", sl->chip_id);
-
-    params = stlink_chipid_get_params(sl->chip_id);
-    if (params) { printf("  dev-type:   %s\n", params->dev_type); }
```

Comment:
I wouldn’t delete this, see :unicorn: below.

Reply:
Yes, I agree - please keep these lines.